R/build_sce.R

```diff
@@ -0,0 +1,91 @@
+#' Build the SCE object from the counts matrix
+#'
+#' @param counts Counts matrix with rownames corresponding to gene names and colnames corresponding to cell barcodes.
+#'   Can be a counts matrix with intron counts specified by -I or an alevin-fry "USA" matrix,
+#'   with intron counts marked by "-U" and ambiguous counts "-A".
+#' @param intron_mode Logical indicating if the files included alignment to intronic regions.
+#'   If TRUE, the main "counts" assay will contain counts aligned to introns and an additional "spliced"
+#'   assay will contain counts aligned to spliced cDNA only.
+#'   Default is FALSE.
+#' @param round_counts Logical indicating in the count matrix should be rounded to integers on import.
+#'   Default is TRUE.
+#'
+#' @return SingleCellExperiment object containing either just a counts assay with spliced cDNA only if
+#'   `intron_mode` is FALSE. If `intron_mode` is TRUE, the counts assay will contain both spliced and unspliced
+#'    counts and the spliced assay will contain the counts for just the spliced cDNA.
+#' @export
+#'
+#' @examples
+#' \dontrun{
+#'
+#' # build a SCE object with only spliced cDNA as the main counts assay
+#' build_sce(counts)
+#'
+#' # build a SCE object with unspliced cDNA as the main counts assay and spliced
+#' # counts as a second assay
+#' build_sce(counts,
+#'           intron_mode = TRUE)
+#'
+#'}
+build_sce <- function(counts,
+                      intron_mode = FALSE,
```

Comment:
Maybe changing this option too?
```suggestion
                      include_unspliced = FALSE,
```